plugin-docker/che-plugin-docker-client/src/test/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfigurationTest.java

```diff
@@ -131,5 +143,98 @@ public void testDockerUriOnNonLinuxMissingProperties() throws Exception {
 
     }
 
+    /**
+     * Check that without property docker host ip from container is DEFAULT_LINUX_DOCKER_HOST_IP
+     */
+    @Test
+    public void testLinuxDefaultDockerHostWithoutBrige() throws Exception {
+        Map<String, String> env = new HashMap<>();
+        NetworkFinder networkFinder = Mockito.mock(NetworkFinder.class);
+        doReturn(Optional.empty()).when(networkFinder).getIPAddress(BRIDGE_LINUX_INTERFACE_NAME);
```

Comment:
It's just that I follow the same approach for stub, mock or real objects.
Also I always use doXXXX() method first so I can return something, create an answer or throw an exception.

Also 'recommended way' is not 'mandatory way'. As long as objects are tested, I'm fine.